src/kernel/messages.c

```diff
@@ -194,13 +194,15 @@ message *msg_message(const char *name, const char *sig, ...)
         }
         else {
             log_error("invalid parameter %s for message type %s\n", paramname, mtype->name);
+            assert(!"invalid parameter for message type");
```

Comment:
Es sollte eigentlich auch alles, was log-level ERROR hat auf stderr erscheinen. Aber ich verstehe natuerlich die Frustration, wenn man ewig so einen Fehler nicht gesehen hat, und nur die Syntax in der XML Datei evtl. falsch war (noch ein Grund fuer simplere Messages). Im Produktionscode will ich das assert aber lieber nicht.